website/content/en/preview/AWS/provisioning.md

```diff
@@ -66,7 +66,7 @@ Select subnets by an arbitrary AWS tag key/value pair:
 Select subnets using wildcards:
 ```
   subnetSelector:
-    Name: *public*
+    Name: "*Public*"

```

Comment:
Is this case-sensitive?

Reply:
Unfortunately yes, this is just an EC2 tag key and value which are [case sensitive](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html).  The eksctl startup documentation results in subnets with a capitalized 'Public' in their names.